### ISSUE TITLE
Related resources

### ISSUE THREAD

**Issue by sdruskat** [sdruskat/software-authorship#40]
### An issue to collect related resources that we may or may not find a better place for

- Mentioned by @danielskatz: https://eos.org/opinions/credit-where-credit-is-due

**Comment by danielskatz** [comment]
The ESIP group that wrote this has been thinking about what roles should be reflected by authorship and which by other means

**Comment by jcolomb** [comment]
Some earlier discussion at FORCE 11: https://demo.hedgedoc.org/WWA2OwbbSeiVXkTkLSwadA.

Additional links from that document:
JATSxml implementation: https://jats4r.org/credit-taxonomy/

Brainhack example: http://brainhack.org/brainhack_jupyter_book/contributors.html

other example with values 1 to 3 to code how much work was done: https://raw.githubusercontent.com/int-brain-lab/paper-behavior/master/authorship/contributions.png
codemetada discussion: https://github.com/codemeta/codemeta/issues/240

first guide on Skilling in general for CitSci projects and data - https://cs4rl.github.io/guide/#/

**Comment by sdruskat** [comment]
Thanks, @jcolomb! Very useful links!

**Comment by jcolomb** [comment]
you are welcome. There is another thing that could be considered: https://www.nature.com/articles/s41467-023-37039-1

Here the idea is to simply "say" what people did, in plain English. It is not very computer readable, but it has the advantage to take the uniqueness of each project into account.

(I personally have not yet made my mind about this strategy)

**Comment by sdruskat** [comment]
> Here the idea is to simply "say" what people did, in plain English. It is not very computer readable, but it has the advantage to take the uniqueness of each project into account.

I think this *could* be a good fallback/additional info, where a taxonomy doesn't work or isn't precise enough.

**Comment by sdruskat** [comment]
From the meeting notes:

- "FAIR Impact project is producing framework for software metrics that is defining Community (effectively a group defined by the goal, e.g. software creation team)/Actor/Roles for research software - we could crosswalk this to the Contribution role taxonomy"

**Comment by sdruskat** [comment]
This is one interesting case (authorship of Linux kernel patches) that we could use to test our work against. Could our definition and roles help solve this/suggest a way this should have gone?

> https://ariel-miculas.github.io/How-I-got-robbed-of-my-first-kernel-contribution/
